scripts/sdk/base-manifest.json

```diff
@@ -2,6 +2,6 @@
     "name": "hydrogen-view-sdk",
     "description": "Embeddable matrix client library, including view components",
     "version": "0.0.4",
-    "main": "./hydrogen.es.js",
+    "main": "./lib-build/hydrogen.es.js",
```

Comment:
The `build.sh` sucks when you `yarn link hydrogen-view-sdk` in another project because it blats the whole `target/` build directory and builds everything from scratch every time which is slow when you're just making small JavaScript changes.

I've gotten around this so far by just running `yarn build:sdk` once, then `yarn run vite build -c vite.sdk-lib-config.js --watch` to keep my bundle up to date.

Reply:
Fixed via https://github.com/vector-im/hydrogen-web/pull/693